network/napalm/napalm_install_config.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+(c) 2016 Elisa Jasinska <elisa@bigwaveit.org>
+    Original prototype by David Barroso <dbarrosop@dravetech.com>
+
+This file is part of Ansible
+
+Ansible is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Ansible is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+DOCUMENTATION = '''
+---
+module: napalm_install_config
+author: "Elisa Jasinska (@fooelisa)"
+version_added: "2.1.0"
+short_description: "Installs the configuration taken from a file on a device supported by NAPALM"
+description:
+    - "This library will take the configuration from a file and load it into a device running any OS supported by napalm.
+      The old configuration will be replaced or merged with the new one."
+requirements:
+    - napalm
+options:
+    hostname:
+        description:
+          - IP or FQDN of the device you want to connect to
+        required: True
+    username:
+        description:
+          - Username
+        required: True
+    password:
+        description:
+          - Password
+        required: True
+    dev_os:
+        description:
+          - OS of the device
+        required: True
+        choices: ['eos', 'junos', 'iosxr', 'fortios', 'ibm', 'ios', 'nxos']
+    timeout:
+        description:
+          - Time in seconds to wait for the device to respond
+        required: False
+        default: 60
+    optional_args:
+        description:
+          - Dictionary of additional arguments passed to underlying driver
+        required: False
+    config_file:
+        description:
+          - Path to the file to load the configuration from
+        required: True
+    commit_changes:
+        description:
+          - If set to True the configuration will be actually merged or replaced. If the set to False,
+            we will not apply the changes, just check and report the diff
+        required: True
+    replace_config:
+        description:
+          - If set to True, the entire configuration on the device will be replaced during the commit.
+            If set to False, we will merge the new config with the existing one. Default- False
+        choices: [yes,on,1,true,no,off,0,false]
```

Comment:
its a bool, make it easier to see it: either `[ yes, no]` or [ true, false ] (not every possible alias provided by ansible)